src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-column.component.html

```diff
@@ -29,19 +29,17 @@
               </label>
             </div>
             <div *ngIf="showExternalLink(interactor)" class="interactorType">
-              <a href="{{externalLink(interactor)}}" target="_blank"
-                 title="{{externalLinkName(interactor)}}">{{ interactor.id }}
-                <i class="icon icon-generic small" data-icon="x"
-                   title="{{externalLinkName(interactor)}}"></i>
+              <a href="{{interactor.identifierLink}}" target="_blank">{{ interactor.identifier }}
```

Comment:
Now that we return the identifier link from the backend, this logic is much simpler. We still need the method `showExternalLink`, but then we can simply use `interactor.identifierLink`, which will have the right link based on the interactor type. The logic to decide which URL to use is in the backend.

I also had to remove the title as we don't have that data, but we could added back if you feel we should have it.